src/app/components/home/home.component.html

```diff
@@ -40,7 +40,10 @@
 
     <div class="row justify-content-center">
       <div class="demo-button mt-3" data-aos="zoom-in-up" data-aos-duration="1000">
-        <button class="btn-grad" routerLink="/login" >Try it out!</button>
+        <div class="row">
+          <button class="btn-grad" routerLink="/demo" >Demo</button>
+          <button class="btn-grad" routerLink="/login" >Pro</button>
```

Comment:
Hey, 
There is nothing like pro in canvasboard. "TRY IT OUT" button will directly go to demo page (i.e demo component)